django/apps/registry.py

```diff
@@ -331,13 +331,19 @@ def unset_installed_apps(self):
         self.apps_ready = self.models_ready = self.ready = True
         self.clear_cache()
 
-    def clear_cache(self):
+    def clear_cache(self, expire_models_cache=False):
```

Comment:
During the app-loading refactor I've tried to eliminate as much as possible methods that did something completely different depending on keyword arguments. Here you're only adding some new logic to clear_cache() hidden behind an `if` clause.

I think model caches should be expired in all cases. For instance, if I remove an app from installed apps, reverse relations to models defined in that app should be cleared from the caches of models defined in other apps. That's why I think you should simply drop the `expire_models_cache` argument and the `if` clause.

If you need to keep them for a reason I missed, can you make a separate method:

    def clear_models_cache(self):
        self.clear_cache()
        self.check_apps_ready()
        for model in self.get_models(include_auto_created=True):
            model._meta._expire_cache()

and replace `apps.clear_cache(expire_models_cache=True)` by `apps.clear_models_cache()`?